data/json/items/containers.json

```diff
@@ -73,7 +73,7 @@
     "looks_like": "30gal_barrel",
     "name": { "str": "steel drum (100L)", "str_pl": "steel drums (100L)" },
     "description": "A huge steel barrel with a watertight lid.",
-    "weight": "12000 g",
+    "weight": "12 kg",
```

Comment:
7.6 kg according to this website. https://www.iecdrums.in/open-mouth-drums.html

Reply:
I may as well go and change it while I'm at it, fair. These numbers aren't from me - I just changed it to kgs instead of gs - but I'll edit it since I'm already working with the code.

Reply:
This will actually need a bunch more changes:
- Aluminum Drum will have its weight adjusted proportionally
- The recipes and disassembly recipes will be adjusted accordingly